### PR TITLE
ao_oss: return actual OSS playing state

### PR DESCRIPTION
**Pull request by rozhuk-im**
fix for: https://github.com/mpv-player/mpv/issues/10640

### CODE REVIEW

File: audio/out/ao_oss.c [hunk 1]
```diff
@@ -48,12 +48,15 @@
 #define AFMT_AC3 -1
 #endif
 
+#ifndef PATH_DEV_DSP
 #define PATH_DEV_DSP "/dev/dsp"
+#endif
+#ifndef PATH_DEV_MIXER
```
Comment (sfan5) [inline audio/out/ao_oss.c:54]: when is this already defined?
it also doesn't seem to have anything to do with the commit title

Comment (rozhuk-im) [reply audio/out/ao_oss.c]: ok, removed.

File: audio/out/ao_oss.c [hunk 2]
```diff
@@ -359,15 +352,15 @@ static void get_state(struct ao *ao, struct mp_pcm_state *state)
         ioctl(p->dsp_fd, SNDCTL_DSP_GETODELAY, &odelay) == -1)
     {
         MP_WARN_IOCTL_ERR(ao);
-        p->playing = false;
         memset(state, 0x00, sizeof(struct mp_pcm_state));
         state->delay = 0.0;
+        state->playing = false;
```
Comment (sfan5) [inline audio/out/ao_oss.c:357]: zero is already false

**Review by sfan5 — APPROVED** [review]
Can't test, code looks good.

**Comment by Dudemanguy** [comment]
@snakedoctr: does this fix ab-loop for you?

**Comment by snakedoctr** [comment]
Sorry, @Dudemanguy.  For some reason github isn't sending me notifications and I never saw this.  Was this commited yet, or do you still need me to test this?  If so, what's the best way to do that now that we're so far past when you submitted this.

**Comment by Dudemanguy** [comment]
It's not committed yet. If you could test this PR real quick and see if it fixes your issue, that would be great.

**Comment by snakedoctr** [comment]
Tested -- works great!

**Comment by Dudemanguy** [comment]
Thanks, I'll pull it in then.

**Pull request merged**